### ISSUE TITLE
browser windows display annoyance

### ISSUE THREAD

**Issue by twillis449** [ska-sa/meqtrees#874]
OK- so I'm testing Meqtrees on both ubuntu 14.04 and 16.04 to test numpy behaviour.  I managed to get a simple script where I pop up four image windows showing test beams to work. This lap top has an intrinsic display of 1600 x 900 pixels.  The bottom two windows are only half displayed. 

![browser_screen_shot](https://cloud.githubusercontent.com/assets/6174767/17410291/d8447c84-5a27-11e6-88ce-2d8568be9d67.png)

I think this is a high-level windows management problem. At one time way back when one could get something like 6 to 8 initial displays in the browser, but the number seems to be shrinking over time. When I 'float' the window the complete contents do appear (including the control GUIs).

**Comment by o-smirnov** [comment]
This is your code, right? It could be that the default size policies of the Qwt plot widgets have changed.

The way the widget layout managers work, they interrogate the widgets about what their minimal size is, and allocate space accordingly.  If the widget says "I can't be squeezed to less than Y pixels in height", then it gets Y pixels no matter what. This is determined by size policy settings: http://doc.qt.io/qt-4.8/qsizepolicy.html#Policy-enum

The "freq/l/m" controls here have a fixed height, and they do take up a lot of real estate. You could look into making them more compact somehow (single line? Smaller fonts? Minimal margins?)

As for the remaining space, you could try to change the "minimal size" of the plot widgets, and play with size policies to allow them to be resized to that minimal size. At the moment it looks like the plot widgets want to stay big.

**Comment by twillis449** [comment]
So what happens when you 'float' a window? When I do that the 'default' size of the pop-up display shrinks quite a bit as the image below shows. I would have thought that according to your argument above that the size should not change.
![floated_window](https://cloud.githubusercontent.com/assets/6174767/17482251/c14ea340-5d35-11e6-9921-1e7f101e0709.png)

**Comment by o-smirnov** [comment]
True, this shows that the plot widget allows itself to be smaller... hmmm, not sure what's going on now!
